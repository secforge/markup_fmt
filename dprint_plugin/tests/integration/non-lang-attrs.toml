# Test custom blocks with non-lang attributes
[vue_custom_block]
default = "lang-attribute"
docs = "none"
