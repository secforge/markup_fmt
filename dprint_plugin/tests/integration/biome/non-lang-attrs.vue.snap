---
source: dprint_plugin/tests/integration.rs
assertion_line: 218
---
<template>
  <div>Test custom blocks with non-lang attributes</div>
</template>

<!-- i18n block with locale attribute -->
<i18n locale="en" lang="json">
{ "hello": "Hello", "goodbye": "Goodbye" }
</i18n>

<!-- i18n block with multiple attributes -->
<i18n lang="json" src="./messages.json" global>
{ "key": "value" }
</i18n>

<!-- docs block with src attribute -->
<docs src="./readme.md">
  <p>Fallback    content</p>
</docs>

<!-- Custom block with custom attributes -->
<metadata
  name="component-meta"
  version="1.0"
>
  <div>Metadata    content</div>
</metadata>
